src/test/java/sirius/db/es/ElasticQuerySpec.groovy

```diff
@@ -8,7 +8,7 @@
 
 package sirius.db.es
 
-
+import com.sun.org.apache.bcel.internal.generic.IAND
```

Comment:
Imports are already fixed in the branch, they don't show atm in this pull request because of githubs current instability